### PR TITLE
Fix #148493 anizle.com

### LINKED ISSUE #148493: anizle.com
### Issue URL (Annoyance)
 [https://anizle.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fanizle.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v4.1.55
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/148493

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by Alex-302** [comment]
Please do not create PRs for issues with `Lowest` priority.

**Comment by gork7777** [comment]
@Alex-302 Why?

**Comment by Alex-302** [comment]
We will decide how to handle them ourselves.

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -282,7 +282,7 @@ havadiskibris.com##img[width="710"][height="80"]
 ||haberkibris.com/banner
 haberkibris.com##img[alt="ads"]
 mynet.com##div[issponsored]
-||anizm.net/js/po-under.js
+/po-under.js$domain=anizle.com|anizm.net
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:285]: Also please add a new domain to https://github.com/AdguardTeam/AdguardFilters/blob/c8a023cfa29a0a4ec1bb23e5c3440612cc01b89e/TurkishFilter/sections/general_extensions.txt#L15

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by gork7777** [comment]
> We will decide how to handle them ourselves.

@Alex-302 There was no such decision before, why did you come to such a decision now?

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@gork7777 To save contributors from aimless work if we decide to close an issue without adding rules.

**Pull request merged**